src/client.rs

```diff
@@ -823,6 +922,12 @@ where
                 .update_delegation(&targets_role, delegation.role(), &raw_signed_meta)
             {
                 Ok(_) => {
+                    /////////////////////////////////////////
+                    // TUF-1.0.9 §5.2.4:
+                    //
+                    //     Persist timestamp metadata. The client MUST write the file to
+                    //     non-volatile storage as FILENAME.EXT (e.g. timestamp.json).
+
```

Comment:
This seems like the wrong snippet? This is for timestamp, while this code is for delegated roles

Reply:
good catch, thanks.